demand_energy.md

```diff
@@ -106,6 +106,40 @@ The non-fuel costs are estimated for some technologies (e.g., light-duty vehicle
 
 
 
+### Direct Air Capture for Carbon Dioxide Removal
+
+GCAM has the capability to model three technologies which consume thermal and/or electrical energy for the sole purpose of removing carbon dioxide from the atmosphere, broadly referred to as direct air capture with carbon storage (DACCS). The first is an aqueous hydroxide solvent process requiring water, high-temperature thermal energy from natural gas for solvent regeneration, and electricity to run ancillary equipment . The second archetype relies on the same aqueous hydroxide solvent process but uses electricity to generate the high temperatures (>900 ˚C) required for regeneration. The third uses solid adsorbents, with the lower-temperature heat required for sorbent regeneration generated by an electric heat pump. These technologies remove CO2 from the atmosphere and send it to geologic storage, along with any captured combustion emissions from process heat.  
```

Comment:
Please add line breaks. Text will be wrapped appropriately when rendered. 
This facilities later edits and tracking.